node/db/src/parity_db.rs

```diff
@@ -33,6 +34,15 @@ impl ParityDbConfig {
     }
 }
 
+impl Default for ParityDbConfig {
+    fn default() -> Self {
+        Self {
+            path: PathBuf::from("/tmp/paritydb"),
```

Comment:
We could make the path an `Option`. Here it would be `None`. In such a case, I would expect the same behaviour as for https://github.com/ChainSafe/forest/blob/f0ac5b50f91eb23a35e1c2e533cb5e65ffe7b542/forest/shared/src/cli/mod.rs#L294-L299